Build/Installer.targets

```diff
@@ -101,10 +101,10 @@
 		A base build should warn if we have 'RemovedSinceLastBase' items to help us remember to clear these out.
 	-->
 	<Target Name="RescuePatching">
-		<!-- <ItemGroup>
-			<RemovedSinceLastBase Include="$(dir-outputBase)/Example.dll" />
+		<ItemGroup>
+			<RemovedSinceLastBase Include="$(dir-outputBase)/DistFiles/Helps/WW-ConceptualIntro/ConceptualIntroduction.htm" />
 		</ItemGroup>
-		<WriteLinesToFile File="%(RemovedSinceLastBase.FullPath)" Lines="" /> -->
+		<WriteLinesToFile File="%(RemovedSinceLastBase.FullPath)" Lines="" />
```

Comment:
What does % mean in MSbuild?

Reply:
Hmm, what is the name of that. 
I think it is an item metadata accessor?
https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-well-known-item-metadata?view=vs-2022